VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts

```diff
@@ -110,7 +110,9 @@ export class JudgeParticipantStatusListComponent implements OnInit {
         const updateParticipantRequest = new UpdateParticipantRequest({
             fullname: this.judge.name,
             display_name: this.judge.display_name,
-            representee: this.judge.representee
+            representee: this.judge.representee,
+            first_name: '',
```

Comment:
This will set the judge's name to be blank. Set this to the judge's current first and last name to ensure we do not accidentaly override values we rely on

`first_name: this.judge.first_name`

Reply:
removed the assignement

Reply:
updated to populate first and last name